### ISSUE TITLE
Content missing after heading/Heading Nesting - SiteImprove - Academic Dates/Events

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#9099]
# Problem

Siteimprove is revealing that the child heading sizing functionality is not working as expected for the academic dates block

https://my2.siteimprove.com/Inspector/998426/A11Y/Page?pageId=29973636376&impmd=512232AC7EB92A0AFBC675A2952A7DCB&issue=cantTell+failed&wcag=twopointtwo#/sia-r78/failed/T0o4bm50czI=;M2Y1MjU5MmU=
 

https://github.com/uiowa/uiowa/blob/main/docroot/modules/custom/uiowa_maui/src/Plugin/Block/AcademicDatesBlock.php#L323

If h2 is selected for the block, returned academic dates should be h3.

**Comment by briand44** [comment]
## Grooming
 
**Quadrant:** 

complicated

 **Discussion:**
 - Maybe we could use headline field and wire that up in order to set heading correctly if not already using.
   - Looks like at least parts of this are there.
   - `docroot/modules/custom/uiowa_maui/src/Form/AcademicDatesForm.php`
   - Needs to be wired up to the child_heading_size arg in `\Drupal\uiowa_maui\Plugin\Block\AcademicDatesBlock::build`

**Proposed solution or definition of done:** 
- Heading structure is output correctly and no longer being flagged as an accessibility issue

**Comment by joewhitsitt** [comment]
Regression caused by hotfix https://github.com/uiowa/uiowa/pull/9330

**Comment by joewhitsitt** [comment]
From the last PR, it looks like you did identify an issue with uiowa_hours which still seems to be an issue.

<img width="998" height="684" alt="Image" src="https://github.com/user-attachments/assets/2fc527a3-5c18-4f21-83bd-f9259e785b81" />
